crates/bevy_pbr/src/material.rs

```diff
@@ -258,12 +262,19 @@ impl<M: SpecializedMaterial> SpecializedMeshPipeline for MaterialPipeline<M> {
 
 impl<M: SpecializedMaterial> FromWorld for MaterialPipeline<M> {
     fn from_world(world: &mut World) -> Self {
-        let asset_server = world.get_resource::<AssetServer>().unwrap();
-        let render_device = world.get_resource::<RenderDevice>().unwrap();
+        let asset_server = world
+            .get_resource::<AssetServer>()
+            .expect("Could not find `AssetServer` in `World`.");
```

Comment:
```suggestion
            .expect("Could not find `AssetServer` resource in `World`.");
```
maybe these should generally state that the thing they could not find is a resource or entity? I won't go through and suggest this change for every instance.